.github/workflows/release_android.yml

```diff
@@ -0,0 +1,91 @@
+name: Release Android APK
+
+on: workflow_dispatch
+
+env:
+  FLUTTER_VERSION: "3.3.1"
+
+jobs:
+  build:
+    runs-on: macos-latest
```

Comment:
I think @klochowicz mentioned it before but I want to ask again nevertheless, why does this have to be `macos-latest`? They are more _expensive_ and sometimes we need to wait unnecessary long to get a slot.

Reply:
I don't know why, but I tested it with an ubuntu machine and the `Configure ndk` step did not work.